src/ansys/mapdl/core/mapdl.py

```diff
@@ -160,6 +161,7 @@ def __init__(
         **start_parm,
     ):
         """Initialize connection with MAPDL."""
+        atexit.register(self.__del__)  # registering to exit properly
```

Comment:
be prepared to run __del__ twice, since atexit might call it once and then the memory cleanup will call it again.  This shouldn't do anything unexpeced

Reply:
Roger. Thank you!